usage-collection/src/main/java/com/sequenceiq/cloudbreak/usage/strategy/CloudwatchUsageProcessingStrategy.java

```diff
@@ -48,8 +53,10 @@ public void processUsage(UsageProto.Event event) {
         CloudwatchRecordRequest recordRequest = CloudwatchRecordRequest.Builder.newBuilder()
                 .withTimestamp(timestamp)
                 .withRawBody(jsonMessageInput)
+                .withMessageBody(event)
+                .withforceRawOutput(true)
```

Comment:
is this typo? should be withForceRawOutput instead of withforceRawOutput

Reply:
Good catch, thanks, it was a typo. Fixed.